website/i18n/zh/docusaurus-plugin-content-docs/current/02-getting-started/quick-add-mtr-test.md

```diff
@@ -0,0 +1,99 @@
+---
+id: quick-add-mtr-test
+sidebar_position: 3.6
+---
+
+# 添加并运行一个mtr测试用例
```

Comment:
The first header is used to render the page title. The following code is recommended instead：
```
# 快速添加 MTR Test

## 添加并运行一个 mtr 测试用例

### ......
```